iChor/iChorCNA_hg19/iChorCNA_hg19.wdl

```diff
@@ -1,327 +1,352 @@
-		String plotFileType = "pdf"
-		String plotYlim
-		File input_gcWig
-		File input_mapWig
-
-		Int memGB = 4
-		Int diskGB = 50
-		Int preemptible = 2
-		Int maxRetries = 1
-
-		String? docker_override
-	}
-
-	command <<<
-
-		Rscript /runIchorCNA.R --id ${sample_id} \
-		--outDir ./ --libdir /ichorCNA \
-		--WIG ${wig_file} \
-		--gcWig  ${input_gcWig} \
-		--mapWig ${input_mapWig} \
-		--normalPanel ${default="None" normalPanel} \
-		--ploidy "${ploidy}" \
-		--normal "${normal}" \
-        ${"--coverage " + mean_depth} \
-		--maxCN ${maxCN} \
-		--includeHOMD ${includeHOMD} \
-		--chrs "${chrs}" \
-		--chrTrain "${chrTrain}" \
-        --chrNormalize "${chrNormalize}" \
-		--genomeStyle "${genomeStyle}" \
-		--genomeBuild "${genomeBuild}" \
-		--estimateNormal ${estimateNormal} \
-		--estimatePloidy ${estimatePloidy}  \
-		--estimateScPrevalence ${estimateClonality} \
-		--scStates "${scStates}" \
-		--centromere ${centromere} \
-		${"--exons.bed " + exons} \
-		--txnE ${txnE} \
-		--txnStrength ${txnStrength} \
-        --minSegmentBins ${minSegmentBins} \
-		--minMapScore ${minMapScore} \
-        --lambdaScaleHyperParam ${lambdaScaleHyperParam} \
-		--fracReadsInChrYForMale ${fracReadsChrYMale} \
-		--maxFracGenomeSubclone ${maxFracGenomeSubclone} \
-        --altFracThreshold ${altFracThreshold} \
-		--maxFracCNASubclone ${maxFracCNASubclone} \
-        --rmCentromereFlankLength ${rmCentromereFlankLength} \
-		--plotFileType ${plotFileType} \
-		--plotYLim "${plotYlim}"
-
-		# Zip optimal solutions
-		mkdir ${sample_id}.optimalSolution
-		cp ${sample_id}/${sample_id}_genomeWide.pdf ${sample_id}.cna.seg ${sample_id}.seg.txt ${sample_id}.seg ${sample_id}.optimalSolution/
-		zip -r ${sample_id}.optimalSolution.zip ${sample_id}.optimalSolution
-        
+    input {
+        File wig_file
+        File? normalPanel
+        String genomeBuild
+        Float? mean_depth
+        String sample_id
+        String ploidy
+        String normal
+        Int maxCN
+        Boolean includeHOMD
+        String chrs
+        String chrTrain
+        String chrNormalize
+        String genomeStyle
+        Boolean estimateNormal
+        Boolean estimatePloidy
+        Boolean estimateClonality
+        String scStates
+        File centromere
+        File? exons
+        Float txnE
+        Int txnStrength
+        Int minSegmentBins
+        Float minMapScore
+        Float fracReadsChrYMale
+        Float maxFracCNASubclone
+        Float maxFracGenomeSubclone
+        Float altFracThreshold
+        Int lambdaScaleHyperParam
+        Int rmCentromereFlankLength
+        String plotFileType = "pdf"
+        String plotYlim
+        File input_gcWig
+        File input_mapWig
+
+        Int memGB = 4
+        Int diskGB = 50
+        Int preemptible = 2
+        Int maxRetries = 1
+
+        String docker_override = "us.gcr.io/tag-team-160914/bloodbiopsy-ichorcna:0.2.1"
+    }
+
+    command <<<
+        set -euxo pipefail
```

Comment:
The `set` additions aren't formatting. However there does not to appear to be any situation in this WDL where it'd not be indicated.